packager/file/http_file.cc

```diff
@@ -288,8 +288,9 @@ void HttpFile::SetupRequest() {
   }
 
   curl_easy_setopt(curl, CURLOPT_URL, url_.c_str());
-  curl_easy_setopt(curl, CURLOPT_USERAGENT,
-                   user_agent_.empty() ? kUserAgent : user_agent_.data());
+  curl_easy_setopt(curl, CURLOPT_USERAGENT, user_agent_.empty() ?
+                   (std::string("ShakaPackager/") + shaka::GetPackagerVersion()).c_str() :
+                   user_agent_.data());
```

Comment:
Can we move it to the constructor?

```c++
if (user_agent_.empty()) {
  user_agent_ += "ShakaPackager/" + shaka::GetPackagerVersion();
}
```

Then the code here can just be: `user_agent_.c_str()`.